legal-api/src/legal_api/resources/v2/business/business_tasks.py

```diff
@@ -189,3 +196,21 @@ def create_incorporate_nr_todo(name_request, order, enabled):
         'enabled': enabled
     }
     return todo
+
+
+def create_conversion_filing_todo(business, order, enabled):
+    """Return a to-do JSON object."""
+    todo = {
+        'task': {
+            'todo': {
+                'business': business.json(),
+                'header': {
+                    'name': 'conversion',
+                    'status': 'NEW'
+                }
+            }
+        },
+        'order': order,
+        'enabled': enabled
+    }
+    return todo
```

Comment:
Oh, this is a neat idea. Is this an alternative to the drop-down staff menu to start a Record Conversion filing? How do those work together?

Reply:
Good point...now that you bring that up...need to confirm.

Reply:
I can't find the UI ticket for the "locked down screen". Do you know it? Could you please add your comment above to it? (Specifically the "non-user-actionable" todo item -- I don't know if we support that atm.)

Reply:
It sounded like some of this should be in the designs that the relationships team already based off of what Linda said last week?